src/integrated/soil_snow_model.jl

```diff
@@ -390,9 +390,17 @@ end
     SoilSublimationwithSnow{FT} <: AbstractSoilSource{FT}
 
 Soil Sublimation source type. Used to defined a method
-of `ClimaLand.source!` for soil sublimation with snow present.
+of `ClimaLand.source!` for soil sublimation with snow present;
+treated implicitly
+in ϑ_l but explicitly in θ_i. This is because we compute

```

Comment:
If sublimation is computed in the implicit tendency, why is this source explicit in `theta_i`?

Reply:
Hmmm so the LHF, SHF, vapor flux, are computed at the same time. so what I intended was for them to be like (e..g. LHF(T_(t+dt), theta_i(t)))

I think this is the case if theta_i does not contribute to the Jacobian. Does that seem right?

Reply:
That sounds right! I think the second part of this comment is confusing - "This is because we compute sublimation and evaporation as part of the boundary flux computation for ϑ_l, which occurs in the implicit tendency."

I'm not sure what would make it more clear, maybe just leaving out that sentence and keeping the first part of the comment

Reply:
okay, I kept the first part only ( and noted we are also implicit in rhoe_int).